x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -15,301 +17,281 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
+      {
+        defaultMessage:
+          'How long to wait for agent connectivity before sending first policy reply, in seconds. Default: 60.',
+      }
+    ),
   },
   {
     key: 'linux.advanced.artifacts.global.base_url',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Base URL from which to download global artifact manifests. Default: https://artifacts.security.elastic.co.',
   },
   {
     key: 'linux.advanced.artifacts.global.manifest_relative_url',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.global.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Relative URL from which to download global artifact manifests. Default: /downloads/endpoint/manifest/artifacts-<version>.zip.',
   },
   {
     key: 'linux.advanced.artifacts.global.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the global artifact manifest signature.',
   },
   {
     key: 'linux.advanced.artifacts.global.interval',
     first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.user.base_url',
-    first_supported_version: '7.11',
-    documentation: '',
-  },
-  {
-    key: 'linux.advanced.artifacts.user.ca_cert',
-    first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between global artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'linux.advanced.artifacts.user.public_key',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Public key used to verify the user artifact manifest signature.',
   },
   {
     key: 'linux.advanced.artifacts.user.interval',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation:
+      'Interval between user artifact manifest download attempts, in seconds. Default: 3600.',
   },
   {
     key: 'linux.advanced.elasticsearch.delay',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Delay for sending events to elasticsearch, in seconds. Default: 120.',
   },
   {
     key: 'linux.advanced.elasticsearch.tls.verify_peer',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation: 'Whether to verify the certificates presented by the peer. Default: true.',
   },
   {
     key: 'linux.advanced.elasticsearch.tls.verify_hostname',
     first_supported_version: '7.11',
-    documentation: 'default is true',
+    documentation:
+      "Whether to verify the hostname of the peer is what's in the certificate. Default: true.",
   },
   {
     key: 'linux.advanced.elasticsearch.tls.ca_cert',
     first_supported_version: '7.11',
-    documentation: '',
+    documentation: 'Text blob of a certificate authority.',
```

Comment:
First supported version is `7.9`.

`PEM encoded certificate for Elasticsearch certificate authority`

@nicholasberlin should we not expose this though and remove it, in favor of the `output.elasticsearch` option you are adding?